### PR TITLE
shared context, spec files and changes in rubocop/gemfile/page object…

### PR DESCRIPTION
**Pull request by anriminasyan**
…s...

### CODE REVIEW

**Review by desislava453 — APPROVED** [review]
OK

**Review by zdravinho — APPROVED** [review]
ok

**Review by AleksandrosV — APPROVED** [review]
OK

**Review by gogstar1899 — CHANGES_REQUESTED** [review]
You should do some changes.

**Review by zdravinho — CHANGES_REQUESTED** [review]
need changes

**Review by okLetsCode — CHANGES_REQUESTED** [review]
Please change files: rubopoc.yml, spec_helper.rb and shared_context_spec.rb

File: .rubocop.yml [hunk 1]
```diff
@@ -8,7 +8,16 @@ require:
   - rubocop-performance
 
 RSpec/HookArgument:
-  EnforcedStyle: each
```
Comment (desislava453) [inline .rubocop.yml:11]: Request change for enforced style.

**Review by AleksandrosV — CHANGES_REQUESTED** [review]
You should do some changes

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -4,15 +4,18 @@
 require 'webdrivers'
 require 'rspec'
 require 'yaml'
+require 'test/unit/assertions'
+include Test::Unit::Assertions
```
Comment (desislava453) [inline spec/spec_helper.rb]: remove the include

File: shared_context/shared_context_spec.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    login.click_sign_in
+    login.type_email(user)
+    login.type_password(pw)
+    login.click_submit
+    sleep(2)
+    expect(@driver.find_element(css: '[style*="padding"]')).to be_displayed       
+  end
+end
+
+RSpec.shared_context 'ban' do
```
Comment (desislava453) [inline shared_context/shared_context_spec.rb]: remove shared context 'ban'

**Review by desislava453 — CHANGES_REQUESTED** [review]
need changes

**Review by desislava453 — APPROVED** [review]
Requested changes are done.

**Review by gogstar1899 — APPROVED** [review]
Looks good now.

**Review by zdravinho — APPROVED** [review]

**Comment by zdravinho** [comment]
ok

**Review by AleksandrosV — APPROVED** [review]
OK

**Review by okLetsCode — APPROVED** [review]
Nice work!

**Pull request merged**